src/Workspaces/Core/Portable/Classification/ClassificationExtensions.cs

```diff
@@ -11,7 +11,7 @@ internal static class ClassificationExtensions
         public static string? GetClassification(this ITypeSymbol type)
             => type.TypeKind switch
             {
-                TypeKind.Class => ClassificationTypeNames.ClassName,
+                TypeKind.Class => type.IsRecord ? ClassificationTypeNames.RecordName : ClassificationTypeNames.ClassName,
```

Comment:
Nit: I think this would be a teeny bit nicer to read
```suggestion
                TypeKind.Class when type.IsRecord => ClassificationTypeNames.RecordName,
                TypeKind.Class => ClassificationTypeNames.ClassName,
```